com.seafile.Client.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Have you submitted this file upstream?

Reply:
https://github.com/haiwen/seafile-client/pull/1388